lib/index.js

```diff
@@ -58,7 +59,18 @@ export default function svgToJS (config) {
     esmx: `import React from 'react'${icons.map(({ titleCase, jsx }) => `\nexport function ${titleCase} (props) {${jsx}}`).join('')}`,
     dts: icons.map(({ camelCase }) => `export declare const ${camelCase}: string`).join('\n'),
     dtsx: icons.map(({ titleCase }) => `export declare const ${titleCase}: React.FunctionComponent<React.SVGProps<SVGElement>>`).join('\n')
-  }
+  };
+
+  // Merge extensions to result if extensions are specified in config
+  [].concat((config.extensions || [])
```

Comment:
Given the context (in config), `extensions` does feel a bit too generic and unclear; I suggest renaming to make the end result more evident. Something in the line of `customOutputs` feels more to the point.